src/server.c

```diff
@@ -5587,10 +5587,10 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "redis_version:%s\r\n", REDIS_VERSION,
             "server_name:%s\r\n", SERVER_NAME,
             "server_version:%s\r\n", SERVER_VERSION,
-            "redis_git_sha1:%s\r\n", serverGitSHA1(),
-            "redis_git_dirty:%i\r\n", strtol(serverGitDirty(),NULL,10) > 0,
-            "redis_build_id:%s\r\n", serverBuildIdString(),
-            "redis_mode:%s\r\n", mode,
+            "server_git_sha1:%s\r\n", serverGitSHA1(),
+            "server_git_dirty:%i\r\n", strtol(serverGitDirty(),NULL,10) > 0,
+            "server_build_id:%s\r\n", serverBuildIdString(),
+            "server_mode:%s\r\n", mode,
```

Comment:
IIRC, we decided to keep "redis_mode" because clients and tools depend on it. It is safe (i.e. acceptable low risk) to change the git sha and build id fields though.

```suggestion
            "redis_mode:%s\r\n", mode,
```

Reply:
I'm getting mixed messages here.  I will revert the changes so that the info fields remain `redis_git_sha1`, `redis_git_dirty`, `redis_build_id`, and `redis_mode`.

Reply:
No, we keep only "redis_mode" and "redis_version". They are checked by clients and tools apparently, so those two have been decided to keep. (@valkey-io/core-team does anyone want to explain exactly why more in detail?)

server_git_sha1, server_git_dirty, server_build_id.

Reply:
On the other hand, @madolson wrote "We should hold off on changing the info fields until we're confident that won't break any applications or workflows." and @PingXie gave it a thumbs up.  I only made any of these changes at the direct request of @hwware